webpack.dev.js

```diff
@@ -7,5 +8,13 @@ module.exports = merge(require('./webpack.common.js'), {
   devServer: {
     contentBase: path.resolve(__dirname, 'public'),
     historyApiFallback: true,
-  }
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('production'),
+        API_URL: JSON.stringify(process.env.API_URL)
```

Comment:
could we maybe hardcode localhost into here instead of putting it in the start script, we're only using the webpack dev config when running locally not building right?